druid-shell/Cargo.toml

```diff
@@ -51,6 +53,8 @@ keyboard-types = { version = "0.5.0", default_features = false }
 
 # Optional dependencies
 image = { version = "0.23.12", optional = true, default_features = false }
+raw-window-handle = { version = "0.3.3", optional = true, default_features = false }
+libc = { version = "0.2.81", optional = true, default_features = false }
```

Comment:
https://github.com/rust-windowing/raw-window-handle/commit/73a41ea28907b0f261816136b1b3f39cfdd791f3

Looks like the dependency on libc was removed a while back?

Reply:
Hm, no cargo release since 2019.

Reply:
https://github.com/rust-windowing/raw-window-handle/pull/67 getting ready for one though.

Reply:
Right, but even 3.3 doesn't need `libc`. Removed.